### PR TITLE
Reading/displaying the linked units (2/2: displaying)

### PR DESCRIPTION
**Pull request by matthias-ronge**
Here comes the second part with the displaying of the linked units.
Please note the explanations in the individual commits. I have tried to better explain why I made the respective changes.

### CODE REVIEW

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/structure.xhtml [hunk 1]
```diff
@@ -17,19 +17,22 @@
...
+                    draggable="true" droppable="true" dragdropScope="DocStructTree">
+                <p:ajax event="select" listener="#{MetadataProcessor.onNodeSelect}"
+                        update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
+                <p:ajax event="dragdrop" listener="#{MetadataProcessor.onNodeDragDrop}"
+                        update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
+                <p:treeNode expandedIcon="ui-icon-document" collapsedIcon="ui-icon-document">
+                    <ui:param name="elementType" value="#{element.type.getNameByLanguage(LanguageForm.locale)}"/>
+                    <h:outputText value="#{elementType}"/> <h:outputText
+                        value=" #{MetadataProcessor.getLogicalPageNumber(element)}" rendered="#{elementType eq msgs.page}"/>
+                    <h:outputText value=" 🔗" rendered="#{MetadataProcessor.isLink(element)}"/>
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/structure.xhtml]: this seems to be a strange value

**Comment by Kathrin-Huber** [comment]
Please don't commit rebase-corrections. Use gits squash to hide this after you made the mistakes.

**Comment by matthias-ronge** [comment]
I have solved the merge conflicts. Is there anything else missing here? Or can you merge the branch now?

**Comment by matthias-ronge** [comment]
This branch doesn’t work (at least also) because auf the bug fixed in PR #2248 − waiting for that to be merged…

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/structure.xhtml (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/structure.xhtml]: You should use fontawesome... Just take random hyperlink icon. But never use such a thing like here!

**Comment by Beacze** [comment]
Could you rebase last 4 codacy commits into one? It looks a bit strange...

**Comment by matthias-ronge** [comment]
Of course I could. The thing is, Codacy always finds new issues when I fix one. That is really annoying. I would like to show that a bit.

Hopefully this is only in the initial phase.

**Comment by Kathrin-Huber** [comment]
You don't need to show your problems with codacy, you should rather hide them!
Squashing the last commits is not a big deal.

**Comment by matthias-ronge** [comment]
This pull request is closed because it is based on a pull request that does not work anymore.

**Pull request closed without merging**